realm/realm-library/src/main/java/io/realm/BaseRealm.java

```diff
@@ -540,16 +539,16 @@ public long getVersion() {
     @ObjectServer
     abstract public RealmResults<Role> getRoles();
 
-    /**
-     * Returns all {@link User} objects available in this Realm. Attach a change listener
-     * using {@link User#addChangeListener(RealmChangeListener)} to be notified about
-     * any future changes.
-     *
-     * @return all roles available in the current Realm.
-     */
-    @Beta
-    @ObjectServer
-    abstract public RealmResults<User> getUsers();
+//    /**
+//     * Returns all users (i.e {@link SyncUser#identity}) available in this Realm. Attach a change listener
+//     * using {@link RealmResults#addChangeListener(RealmChangeListener)} to be notified about
+//     * any future changes.
+//     *
+//     * @return all roles available in the current Realm.
+//     */
+//    @Beta
+//    @ObjectServer
+//    abstract public RealmResults<String> getPermissionUsers();
```

Comment:
`RealmResults<String>`? Subtable queries are coming? 😮

Reply:
Not yet unfortunately.